packages/components/popover/src/PopoverClose.tsx

```diff
@@ -0,0 +1,42 @@
+import * as RadixPopover from '@radix-ui/react-popover'
+import { Icon } from '@spark-ui/icon'
+import { IconButton } from '@spark-ui/icon-button'
+import { Close as CloseSVG } from '@spark-ui/icons/dist/icons/Close'
+import { cx } from 'class-variance-authority'
+import { forwardRef, useEffect } from 'react'
+
+import { usePopover } from './PopoverContext'
+
+export type CloseProps = RadixPopover.PopoverCloseProps & {
+  'aria-label': string
```

Comment:
Why do we need to add this to type? Isn't it already defined by default?

Reply:
I need to make it mandatory for a11y

Reply:
Didn't we agree yesterday to avoid having that attribute mandatory?

Reply:
Ok then I'll update the doc too